op-batcher/batcher/driver.go

```diff
@@ -215,6 +216,14 @@ func (l *BatchSubmitter) StartBatchSubmitting() error {
 			return fmt.Errorf("could not register with batch inbox contract: %w", err)
 		}
 
+		l.submitter = NewEspressoTransactionSubmitter(
+			WithContext(l.shutdownCtx),
+			WithWaitGroup(l.wg),
+			WithEspressoClient(l.Espresso),
+		)
+		l.submitter.SpawnWorkers(4, 4)
```

Comment:
Would it be worth create some configuration variable for these values? Is there any advantage to have different numbers here, that is why not use the same number of SubmitTransaction/VerifyReceipt  workers?

Reply:
I originally had the number of workers match across both.  I suppose having them separate allows for you to tune these separately.  This can be beneficial, especially when it comes to the verify receipt workers, as they can end up paused and asleep when re-attempting  a job that hasn't been verified yet.  So in that case you might want more receipt verify workers in order to try and ensure that we are are still making some progress even while a few of our workers are sleeping.

Reply:
I see, thanks.